contrib/src/test/java/org/apache/apex/malhar/contrib/kudu/KuduCreateUpdateDeleteOutputOperatorTest.java

```diff
@@ -0,0 +1,336 @@
+/**
```

Comment:
no. I was told that the tests are explicitly disabled in the contrib packages and hence this should not effect ? The default implementation looks for a property file name to use the host names that it will use to connect to. The java doc of the operator gives the file name as well as the property key names.